src/index.html

```diff
@@ -5,8 +5,401 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- Nav -->
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#lapcomp" class="nav__link">Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Catalog!!! -->
+    <main>
+      <div class="card" data-qa="card">
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__quality">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__pword">
+            Price:
+          </p>
+          <p class="card__pcost">
+            $2,199
+          </p>
+        </div>
+        <a
+          class="card__button"
+          href="#"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card" data-qa="card">
```

Comment:
Add a blank line between the cards
```suggestion
      </div>
      
      <div class="card" data-qa="card">
```